src/systems/force_torque/ForceTorque.cc

```diff
@@ -129,6 +129,12 @@ ForceTorque::ForceTorque()
 //////////////////////////////////////////////////
 ForceTorque::~ForceTorque() = default;
 
+//////////////////////////////////////////////////
+System::PriorityType ForceTorque::ConfigurePriority()
+{
+  return 10;
```

Comment:
Instead of hardcoding the priority here, can we add a header file with a handful of well defined priorities
e.g.
```
constexpr System::PriorityType kPrePhysicsPriority = 0;
constexpr System::PriorityType kPhysicsPriority = 10;
constexpr System::PriorityType kPostPhysicsPriority = 100;
```
and use that instead? I think that will convey the intent of setting the default priority here better. I'd also add a default priority for the PhysicsSystem to go with this change.

Reply:
We probably should so the same for the `UserCommandSystem`.

Reply:
I've added SystemPriorityConstants.hh in https://github.com/gazebosim/gz-sim/pull/2500/commits/20542139b2037ae0f108864d031267e64108e6e8 and used those constants to configure priority of the Physics and ForceTorque systems.

I haven't touched `UserCommandSystem` yet; I want to make sure the `Physics` change works properly first

Reply:
that broke the windows build; I'll fix shortly...

Reply:
inline namespace removed in https://github.com/gazebosim/gz-sim/pull/2500/commits/d32c9d693bc2d56a9be88d7c64730f8bffe2b079

Reply:
updated user commands system priority in https://github.com/gazebosim/gz-sim/pull/2500/commits/735dd7dcd9f8643126d5d72d7898fc34123b1a4c